src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -2018,7 +1884,7 @@ static void Main()
 }
 ";
             var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
-            CompileAndVerify(comp, expectedOutput: @"Span").VerifyDiagnostics();
+            CompileAndVerify(comp, expectedOutput: @"ReadOnlySpan", verify: Verification.Fails).VerifyDiagnostics();
```

Comment:
Why was IL verification succeeding with `Span` but now fails with `ReadOnlySpan` overload and new natural type?